Project/API/src/src/main/java/com/example/models/EmbededAssetModel.java

```diff
@@ -0,0 +1,28 @@
+package com.example.models;
+
+import org.bson.types.ObjectId;
+import org.springframework.data.annotation.Id;
+import org.springframework.data.mongodb.core.index.Indexed;
+import org.springframework.data.mongodb.core.mapping.Document;
+
+import com.mongodb.lang.NonNull;
+
+import jakarta.validation.constraints.Max;
+import jakarta.validation.constraints.Min;
+import jakarta.validation.constraints.NotNull;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Document(collection = "Embeded_Asset")
```

Comment:
Mongo collection name should be simple (as the best practice) embeded_assets like this